File: Warmup1_JAVA/nearHundred.java
/*
PROMPT
Given an int n, return true if it is within 10 of 100 or 200.
Note: Math.abs(num) computes the absolute value of a number.
*/

public boolean nearHundred(int n) {
  return ((Math.abs(100-n) <= 10) || (Math.abs(200-n) <= 10));
}

/*
This return statement returns true if either of the expressions
are true.
This is checking the values surrounding both 100 and 200 to see
if n is within 10 points of either.
Can also be written...
---
if (Math.abs(100-n) <= 10) {
  return true;
}
else if (Math.abs(200-n) <= 10) {
  return true;
}
return false;
*/

File: Warmup1_PYTHON/near_hundred.py
'''
PROMPT
Given an int n, return True if it is within 10 of
100 or 200. Note: abs(num) computes the absolute
value of a number.
'''

def near_hundred(n):
  return ((abs(100 - n) <= 10) or (abs(200 - n) <= 10))

'''
This return statement returns True if either of the expressions
are true.
This is checking the values surrounding both 100 and 200 to see
if n is within 10 points of either.
Can also be written...
---
if (abs(100-n) <= 10):
  return True
elif (abs(200-n) <= 10):
  return True
return False
'''
